test/cases/api/common/aws.sh

```diff
@@ -17,11 +17,15 @@ function installClient() {
   $AWS_CMD --version
 
   if ! hash govc; then
```

Comment:
My god, I forgot we are reusing the same code for testing aws and vsphere. I wonder if we can split that so we don't need to install `govc` every time we test aws images. :thinking: Just food for thought.

Reply:
Makes sense, I was wondering what `govc` is doing in `aws.sh`

Reply:
I am fixing this in a separate PR, but would like to get an initial review.